XPNet.CLR/Data/GeneratedDataRefs/Sim/Aircraft/ArtstabilityDataRefs.cs

```diff
@@ -0,0 +1,16 @@
+using System;
```

Comment:
I generally agree, I just wasn't comfortable enough with the build process to pull it off in this PR. Would love the assistance. The only issue we'll need to resolve is the top level reference in the non generated code to the Sim class.

Reply:
Ok, since we have an agreement in principal then I'm going to resolve this discussion, and I'll either merge all this after the build changes go in, or I'll merge this to master first and then add the auto-build as soon as the build changes go in.